### PR TITLE
Expose Tether event system as imperative and props API

### PR DESCRIPTION
**Pull request by slorber**
handles
https://github.com/souporserious/react-tether/issues/28
https://github.com/souporserious/react-tether/issues/29

### CODE REVIEW

File: src/TetherComponent.jsx [hunk 1]
```diff
@@ -75,18 +78,43 @@ class TetherComponent extends Component {
   componentWillUnmount() {
     this._destroy()
   }
-
+  
+  getTether() {
```
Comment (souporserious) [inline src/TetherComponent.jsx]: how do you feel about calling this `getTetherInstance`?

File: src/TetherComponent.jsx [hunk 2]
```diff
@@ -75,18 +78,43 @@ class TetherComponent extends Component {
...
   disable() {
     this._tether.disable()
   }
 
   enable() {
     this._tether.enable()
   }
+  
+  on(event, handler, ctx) {
+    this._tether.on(event,handler,ctx);
```
Comment (souporserious) [inline src/TetherComponent.jsx]: nitpicking, but can we can get spaces between the arguments here

File: src/TetherComponent.jsx [hunk 3]
```diff
@@ -75,18 +78,43 @@ class TetherComponent extends Component {
...
+  off(event, handler) {
+    this._tether.off(event,handler)
+  }
 
   position() {
     this._tether.position()
   }
+  
+  _registerEventListeners() {
+    if ( this.props.onUpdate ) {
```
Comment (souporserious) [inline src/TetherComponent.jsx]: another nitpick, but can we please get the spaces inside the parens removed from here as well as the other to match the original code style.

File: src/TetherComponent.jsx (hunk 1 above)
Comment (slorber) [inline src/TetherComponent.jsx]: yes, was thinking of using that name too

**Comment by slorber** [comment]
I've made the style changes and renamed getTether.

(didn't run npm install before commit, i guess you'll want to run it yourself before releasing)

**Comment by slorber** [comment]
@souporserious tell me if you want to change anything else :)

**Comment by souporserious** [comment]
Thanks for cleaning that up!

**Comment by slorber** [comment]
(don't forget to run npm install before release :))

**Comment by slorber** [comment]
@souporserious can you publish the 0.5.3 release plz ? :)

**Pull request merged**